### PR TITLE
Höganäs, Sweden

### PR DESCRIPTION
**Pull request by sergiyprotsiv**
Municipality of Höganäs, Sweden.

### CODE REVIEW

File: sources/se/municipality_of_hoganas.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "coverage": {
+        "country": "se",
+        "city": "Höganäs"
+    },
+    "data": "http://maps.hoganas.se/arcgis/rest/services/Baskarta/webb_baskarta_nedtonad/MapServer/0",
+    "type": "ESRI",
+    "conform": {
+        "type": "geojson",
+        "number": "AddressPlace",
+        "street": "StreetName",
```
Comment (thatdatabaseguy) [inline sources/se/municipality_of_hoganas.json]: These regexes should do it:

```json
"number": {
    "function": "regexp",
    "field": "Name",
    "pattern": "^(?:.*?)\\s*([\\d]+\\w*(?:\\-[\\d]+\\w*)?)$"
},
"street": {
    "function": "regexp",
    "field": "Name",
    "pattern": "^(.*?)\\s*(?:[\\d]+\\w*(?:\\-[\\d]+\\w*)?)$"
},
```

**Comment by thatdatabaseguy** [comment]
@sergiyprotsiv maybe try merging in master and re-running? I had that same issue happen and it had to do with schema validation. The NodeJS validation script tries to parse and validate regexes, and it uses a different regex flavor than Python, so in latest master we just treat the regexes as strings for schema validation purposes.

**Comment by iandees** [comment]
This might be related to me merging in a very old branch...

**Comment by iandees** [comment]
I am confused here. When I did a `git bisect`, the tests pass until 5be27b2d, when I merged in #1761. That pull request doesn't seem to have anything wrong with it, though.

**Comment by thatdatabaseguy** [comment]
Re-run this please.

**Comment by thatdatabaseguy** [comment]
Fixed that merge issue, but will have to pull in upstream master for the Travis build to pass.

**Comment by sergiyprotsiv** [comment]
Ok, can you let me know exactly what to do to pull in upstream master, please?

**Comment by thatdatabaseguy** [comment]
Should be (from the se-hgs branch in your fork):

```shell
git pull https://github.com/openaddresses/openaddresses master
```

That will create a default merge commit, then ```git push``` and it will show up here.

**Comment by sergiyprotsiv** [comment]
Ok, thanks, that worked. Did the same for Aargau

**Pull request merged**